### PR TITLE
fix(cli-e2e): make it great again

### PR DESCRIPTION
**Pull request by naorpeled**
# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests
- [x] Other

# Why is it needed?

addresses [this task](https://github.com/orgs/qwikifiers/projects/1/views/1?pane=issue&itemId=65262666).

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/qwikifiers/qwik-ui/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [ ] I have ran `pnpm change` and documented my changes
- [ ] I have add necessary docs (if needed)
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

**Comment by maiieul** [comment]
Thanks @naorpeled! Is this still a WIP?

**Comment by naorpeled** [comment]
> Thanks @naorpeled! Is this still a WIP?

Yes, planning on finalizing today/tomorrow. Will let you know when it's ready

File: packages/cli-e2e/src/cli.smoke.spec.ts [hunk 1]
```diff
@@ -23,13 +23,18 @@ describe('Qwik UI CLI Smoke test', () => {
...
         env: process.env,
+        stdio: 'inherit',
       },
     );
+    execSync('npx -y qwik-ui@e2e add button', {
+      cwd: projectDirectory,
+      env: process.env,
+      stdio: 'inherit',
+    });
     const buttonIsInTheRightPlace = existsSync(
```
Comment (naorpeled) [inline packages/cli-e2e/src/cli.smoke.spec.ts:38]: @shairez 
in a separate PR I'll remove the code that hints that it's possible to provide component names in the `init` phase 🙏

Comment (shairez) [reply packages/cli-e2e/src/cli.smoke.spec.ts:38]: but this is not the init phase no?

Comment (naorpeled) [reply packages/cli-e2e/src/cli.smoke.spec.ts:38]: It's not, but in the test, before my change, we had the components flag with the init command

**Comment by naorpeled** [comment]
Hey,
@maiieul and @shairez 
can we get this merged in? 🙏

**Review by maiieul — APPROVED** [review]
LGTM! But I might be lacking a bit of experience on this. @shairez can you have a quick look too?

**Comment by maiieul** [comment]
@naorpeled sorry took this long, it's also because CI was broken for some time ^^

**Comment by naorpeled** [comment]
> @naorpeled sorry took this long, it's also because CI was broken for some time ^^

All gucci bud!
after this will be merged, I'll be work on the next PR that is related to the CLI :)

**Comment by shairez** [comment]
Sorry for the delay @naorpeled !

Thank you very much for this 🙏 ❤️

**Pull request merged**